apps/api/serializers.py

```diff
@@ -197,3 +197,5 @@ class TriggerBotMessageRequest(serializers.Serializer):
     experiment = serializers.UUIDField(label="Experiment ID")
     prompt_text = serializers.CharField(label="Prompt to go to bot")
     start_new_session = serializers.BooleanField(label="Starts a new session", required=False, default=False)
+    session_data = serializers.DictField(help_text="Update session data", required=False, default=dict)
+    participant_data = serializers.DictField(help_text="Update Participant Data", required=False, default=dict)
```

Comment:
Add note about data being merged:
```suggestion
    session_data = serializers.DictField(help_text="Update session data. This will be merged with existing data.", required=False, default=dict)
    participant_data = serializers.DictField(help_text="Update participant data. This will be merged with existing data.", required=False, default=dict)
```